graql/reasoner/atom/binary/AttributeAtom.java

```diff
@@ -208,10 +214,26 @@ public final boolean equals(Object obj) {
     }
 
     @Override
-    boolean isBaseEquivalent(Object obj){
-        if (!super.isBaseEquivalent(obj)) return false;
+    public boolean isAlphaEquivalent(Object obj) {
```

Comment:
why do we have new equivalences on AttributeAtom?

Reply:
We don't. We just call everything explicitly.

Reply:
ok cool, i think this is more readable :)